x-pack/plugin/monitoring/src/main/java/org/elasticsearch/xpack/monitoring/MonitoringTemplateRegistry.java

```diff
@@ -78,7 +78,7 @@ public class MonitoringTemplateRegistry extends IndexTemplateRegistry {
      * writes monitoring data in ECS format as of 8.0. These templates define the ECS schema as well as alias fields for the old monitoring
      * mappings that point to the corresponding ECS fields.
      */
-    public static final int STACK_MONITORING_REGISTRY_VERSION = Version.V_8_0_0.id + 2;
+    public static final int STACK_MONITORING_REGISTRY_VERSION = Version.V_8_0_0.id + 3;
```

Comment:
what does this 3 (previously 2) mean?

Reply:
In an upgrade scenario, it allows the registry to determine whether the templates are different from the ones already installed in order to update them